modules/@apostrophecms/ui/ui/apos/mixins/AposPublishMixin.js

```diff
@@ -65,9 +65,10 @@ export default {
               // Retry now that ancestors are published
               return this.publish(doc);
             } catch (e) {
+              const errorMessage = e.message === 'forbidden' ? 'apostrophe:errorWhilePublishingParentPageForbidden' : e.message;
```

Comment:
Use `e.name`, should also be `forbidden`, while the message could be different if someone takes it upon themselves to throw a more informative one on the server side so that could break one day